index.html

```diff
@@ -29,21 +29,21 @@ <h2 class="player-1">Player 1</h2>
           <header class="winner-declaration">
```

Comment:
On line 21 you could use an "aside" instead of a section for some slightly more semantic html